### PR TITLE
Feat/rework sprint

### PR DESCRIPTION
**Pull request by rs-amp**

### CODE REVIEW

File: src/components/FilterBar.tsx [hunk 1]
```diff
@@ -392,17 +440,65 @@ const FilterBar = ({
...
-            <img alt="filter icon" src={FilterIcon} />
-          </IconButton>
+          <Box>
+            <IconButton
+              className={classes.basketBtn}
+              aria-label="toggle"
+              size="small"
+              onClick={() => {
+                dispatch(getContentItems(locale, 1, filter));
+                // setSelectedContent([]);
```
Comment (rs-amp) [inline src/components/FilterBar.tsx]: Remove?

File: src/components/FilterBar.tsx [hunk 2]
```diff
@@ -423,7 +519,7 @@ const FilterBar = ({
             <div className={classes.popover}>
               <div className={classes.filterHeading}>
                 <TextField
-                  placeholder="Search"
+                  placeholder="Content item name"
```
Comment (rs-amp) [inline src/components/FilterBar.tsx]: "Label" might be more accurate

File: src/store/loadings/loadProgress.ts [hunk 3]
```diff
@@ -0,0 +1,248 @@
+import { AppDispatch } from '../store';
+import { LoadModal } from './loadModal';
+import { setDialogLoader } from './loadings.actions';
+
+export enum SubmissionStage {}
```
Comment (rs-amp) [inline src/store/loadings/loadProgress.ts]: Unused

File: src/store/loadings/loadProgress.ts [hunk 4]
```diff
@@ -0,0 +1,248 @@
...
+
+export enum ImportStage {}
```
Comment (rs-amp) [inline src/store/loadings/loadProgress.ts]: Unused

File: src/components/Submissions.tsx [hunk 5]
```diff
@@ -68,13 +136,35 @@ const Submissions = (props) => {
...
+              </span>
+            ))}
+        </>
+      ),
+    },
+    {
+      id: 'created_at',
+      label: 'Creation Date',
+      format: (timestamp: number) =>
+        new Date(timestamp).toLocaleDateString().replaceAll('.', '/'),
```
Comment (rs-amp) [inline src/components/Submissions.tsx:166]: Weird conversion here. If you want a specific date format, maybe pass your own date format string?

File: README.md [hunk 6]
```diff
@@ -1,9 +1,26 @@
...
+## Building
+This extension was developed and tested using:
+- npm 9.x
+- node 16.x
+
+To switch to the correct node version it is recommended to have [Node Version Manager](https://github.com/nvm-sh/nvm) installed. Commands:
+
+nvm install 16
+
+nvm use 16
```
Comment (rs-amp) [inline README.md:15]: Could use some comments about building (`npm run build`) and live development (`npm run start`).

File: docs/CONFIGURATION.md [hunk 7]
```diff
@@ -154,3 +157,14 @@ When configuring content types for translation, this uses  JSONPath for defining
 | sourceLocale<br /><br />*(optional)*           | string   | The original locale of the content item, which the submitted items will be translated from. This locale must be configured within your Dynamic Content hub.<br /><br />The source locale is constructed from a mandatory language code ([ISO  639-1:2002](https://www.iso.org/standard/22109.html)), and an optional country code ([ISO  3166-2:2013](https://www.iso.org/standard/63546.html)). |
 | targetLocales<br /><br />*(optional)*          | array    | One or more locales, which the submitted items will be translated to. These locales must be configured within your Dynamic Content hub.<br /><br />Destination locales are constructed from a mandatory language code ([ISO  639-1:2002](https://www.iso.org/standard/22109.html)), and an optional country code ([ISO  3166-2:2013](https://www.iso.org/standard/63546.html)). |
 | additionalInstructions<br /><br />*(optional)* | string   | Any additional instructions to send along with the submission. |
+
+
+### Delivery Keys
+You are free to use Delivery Keys but please be aware of the following:
```
Comment (rs-amp) [inline docs/CONFIGURATION.md]:
```suggestion
You are free to use Delivery Keys, but please be aware of the following:

```

File: docs/CONFIGURATION.md [hunk 8]
```diff
@@ -154,3 +157,14 @@ When configuring content types for translation, this uses  JSONPath for defining
...
 | targetLocales<br /><br />*(optional)*          | array    | One or more locales, which the submitted items will be translated to. These locales must be configured within your Dynamic Content hub.<br /><br />Destination locales are constructed from a mandatory language code ([ISO  639-1:2002](https://www.iso.org/standard/22109.html)), and an optional country code ([ISO  3166-2:2013](https://www.iso.org/standard/63546.html)). |
 | additionalInstructions<br /><br />*(optional)* | string   | Any additional instructions to send along with the submission. |
+
+
+### Delivery Keys
+You are free to use Delivery Keys but please be aware of the following:
+
+- Don't set Delivery Keys as Mandatory or translations will fail
+- If you define a Delivery Key format in the Schema eg content/* after you have already created content, you'll need to sync to enable the translations to proceed
+- If using Regexp, use prefix as _locale which is used as a suffix for translations
```
Comment (rs-amp) [inline docs/CONFIGURATION.md]:
```suggestion
- If using a regular expression to validate your delivery key, your pattern should allow for a suffix, as localised items automatically generate with delivery keys suffixed with `_<locale>`, eg. `_en-GB`. If your delivery key pattern only requires a suffix, such as `content/`, then it should work without issue.
```

File: docs/USAGE.md [hunk 9]
```diff
@@ -43,17 +44,17 @@ Some values within the GlobalLink Connect dashboard are configurable at the poin
 When you create your content, there are a couple of things you will need to do to ensure that this content is available for translation:
 
 1. Assign the source locale to your content item.
-   - For example, if you are looking to translate your content from English to German, you may want to assign en-US, or en-GB.
-2. (Optionally) assign the content item to a person.
-   - This is not mandatory for translation but may help with finding the content item when you create a submission, as content items can be filtered by assignee.
+   - For example, if you are looking to translate your content from English to German, you may want to assign en-US.
+2. (Optionally) assign the content item to a person/s.
```
Comment (rs-amp) [inline docs/USAGE.md]: Odd, `a persons` doesn't make much sense. Could replace with "someone" if you want it to be more ambiguous that you can assign more than one person.

File: docs/CONFIGURATION.md (hunk 8 above)
Comment (rs-amp) [reply docs/CONFIGURATION.md]: This is also explained in USAGE.md.

**Review by rs-amp — COMMENTED** [review]
`getCountryCode` seems to be redefined a lot, perhaps this could be put in a utils file.

**Review by neilmistryamplience — APPROVED** [review]
approved for testing production branch

**Review by scottstore4 — APPROVED** [review]

**Pull request merged**